### PR TITLE
Bump Prow images

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Bump Prow images
- Needed for ok-to-test fix

**Related issue(s)**

See #403

### CODE REVIEW

**Comment by michal-hudy** [comment]
How about using newer version of Prow? They introduced new feature:
>February 13, 2019 prow (both plank and crier) can set status on the commit for postsubmit jobs on github now! Type of jobs can be reported to github is gated by a config field like
  ```yaml
  github_reporter: 
    job_types_to_report:
    - presubmit
    - postsubmit
  ```
  >now and default to report for presubmit only. **The default will change in April to include postsubmit jobs as well** You can also add skip_report: true to your post-submit jobs to skip reporting if you enable postsubmit reporting on.

**Comment by suleymanakbas91** [comment]
I realized that usage of these images have been reverted in kubernetes/test-infra: https://github.com/kubernetes/test-infra/pull/11205
I'll wait until they bump the images to a stable version.

**Comment by michal-hudy** [comment]
@suleymanakbas91 I see that https://github.com/kubernetes/test-infra/issues/11430 has been closed. So we can update Prow. What do you think? :smile:

**Comment by tehcyx** [comment]
As part of #148, with PR #747 I tested upgrading to a release from yesterday, which worked for my tests:
https://github.com/tehcyx/test-infra/blob/cafd60993e0c767fd2b6350177359dc9c22bdc2b/prow/config.yaml#L370-L373

https://github.com/tehcyx/test-infra/blob/cafd60993e0c767fd2b6350177359dc9c22bdc2b/prow/cluster/starter.yaml#L177

**Comment by suleymanakbas91** [comment]
@michal-hudy Yes, I already started testing again using the image version @tehcyx said.

File: prow/config.yaml [hunk 1]
```diff
@@ -307,9 +307,12 @@ deck:
...
     viewers:
-      "started.json|finished.json": ["metadata-viewer"]
-      "build-log.txt": ["build-log-viewer"]
-      "artifacts/junit.*\\.xml": ["junit-viewer"]
+      "started.json|finished.json":
+      - "metadata"
+      "build-log.txt":
+      - "buildlog"
+      "artifacts/junit.*\\.xml":
+      - "junit"
```
Comment (michal-hudy) [inline prow/config.yaml]: Is it needed? We don't have `junit` reports.

**Pull request merged**